### ISSUE TITLE
Default breakpoints sm, md, lg are off by 1px

### ISSUE THREAD

**Issue by slackday** [kettanaito/atomic-layout#204]
## What:

Defaults are defined as

```
  breakpoints: {
    xs: {
      maxWidth: '575px',
    },
    sm: {
      minWidth: '576px',
      maxWidth: '768px',
    },
    md: {
      minWidth: '769px',
      maxWidth: '992px',
    },
    lg: {
      minWidth: '993px',
      maxWidth: '1199px',
    },
    xl: {
      minWidth: '1200px',
    },
  },
```

should be 

```
  breakpoints: {
    xs: {
      maxWidth: '575px',
    },
    sm: {
      minWidth: '576px',
      maxWidth: '767px',
    },
    md: {
      minWidth: '768px',
      maxWidth: '991px',
    },
    lg: {
      minWidth: '992px',
      maxWidth: '1199px',
    },
    xl: {
      minWidth: '1200px',
    },
  },
```

https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints

**Comment by kettanaito** [comment]
Hello, @slackday. Thanks for reporting this. Indeed, you appear to be right, default breakpoints have 1 px off from the Bootstrap ones. I'll see your pull request and let's fix that together.

**Comment by kettanaito** [comment]
The fix is published in `0.9.4`.

Thank you for the help and welcome to contributors!
